DESCRIPTION

```diff
@@ -1,6 +1,6 @@
 Package: QFeatures
 Title: Quantitative features for mass spectrometry data
-Version: 1.3.3
+Version: 1.3.4
```

Comment:
The version should stay 1.3.3, which corresponds to updates specific to github development, that will be pushed to Bioc devel (currently at 1.3.2).